helios-services/src/main/java/com/spotify/helios/agent/TaskRunner.java

```diff
@@ -77,7 +77,7 @@ private TaskRunner(final Builder builder) {
     this.listener = checkNotNull(builder.listener, "listener");
     this.existingContainerId = builder.existingContainerId;
     this.registrar = checkNotNull(builder.registrar, "registrar");
-    this.secondsToWaitBeforeKill = checkNotNull(builder.secondsToWaitBeforeKill, "waitBeforeKill");
+    this.secondsToWaitBeforeKill = builder.secondsToWaitBeforeKill;
```

Comment:
`builder.secondsToWaitBeforeKill` is a primitive type and therefore `checkNotNull` was pointless